README.textile

```diff
@@ -114,6 +115,66 @@ installation and configuration. Information about reporting problems and
 getting help for ImageMagick is available at the "ImageMagick web site":http://www.imagemagick.org
 or the "ImageMagick Forum":http://www.imagemagick.org/discourse-server.
 
+h2(#dev-setup). Development Setup
+
+In order to minimize issues on your local machine, we recommend that you make
+use of a "Vagrant installation":https://github.com/tjschuck/rake-compiler-dev-box.
+
+Steps to get up and running with a passing build are as follows:
+
+h4. 1) set up the Vagrant environment
+
+<pre>
+git clone https://github.com/tjschuck/rake-compiler-dev-box.git
+cd rake-compiler-dev-box
+vagrant up
+</pre>
+
+This last part will probably take a while as it has to download an Ubuntu image
+and configure it. If there is an error during this process, you may need to
+reboot your computer and enable virtualization in your BIOS settings. You may
+also want to add the following line to the Vagrantfile for convenience:
+
+<pre>
+config.ssh.forward_agent = true
+</pre>
+
+You'll need to run `vagrant reload` for the settings to take effect.
+
+h4. 2) log in to the vagrant box and clone RMagick
+
+<pre>
+vagrant ssh
+git clone https://github.com/gemhome/rmagick.git # or your fork
```

Comment:
@bf4 Ah, okay. When I ran `vagrant ssh` it put me in the home directory and when I didn't see the project I assumed I didn't have access to it. Should have read the docs telling that it's in `/vagrant`.